### PR TITLE
Added host, port (with default value) variables using cli program mode

### PR DESCRIPTION
**Pull request by vladdancer**

### CODE REVIEW

File: pub_weight/src/main.rs [hunk 1]
```diff
@@ -14,10 +15,28 @@ fn generate_client_id() -> String {
...
+          .index(2)
+          .help("Port of the MQTT Server")
+        )
+        .get_matches();
 
-    println!("Connecting to {:?} ... ", server_addr);
-    let mut stream = TcpStream::connect(server_addr).unwrap();
+    let server_addr = matches.value_of("host").unwrap().to_string();
+    let server_port = matches.value_of("port").unwrap_or("1883").to_string();
+    let server_host = [server_addr, ":".to_string(), server_port].concat();
```
Comment (vladbat00) [inline pub_weight/src/main.rs]:
```suggestion:-3+0
    let server_addr = matches.value_of("host").unwrap();
    let server_port = matches.value_of("port").unwrap_or("1883");
    let server_host = format!("{}:{}", server_addr, server_port);
```

Comment (vladbat00) [reply pub_weight/src/main.rs]: Multi-line suggestions don't work :(

**Comment by vladbat00** [comment]
1 more advise: install and run rustfmt :)
```bash
rustup component add rustfmt
cargo fmt
```

And even better with clippy!
```bash
rustup component add clippy
cargo clippy
```

**Comment by vladdancer** [comment]
Updated.

**Pull request merged**